docs/src/tutorial/add-remove-nodes.md

```diff
@@ -0,0 +1,116 @@
+# Adding and Removing Nodes
+
+## What you will need
+- Multipass (See [Multipass Installation](https://multipass.run/install))
+- A bootstrapped cluster using Canonical K8s (See (getting-started.md)[Getting Started]
+- The `dns` and `network` components enabled on your cluster.
+
+### 1. Verify the Status Of Your Cluster
+
+Verify the status of your cluster with:
+
+```
+sudo k8s status
+```
+
+Make sure you can see `k8s is ready.` in the output of the command.
+
+### 2. Start a Multipass VM
+
+Our second node will live on Ubuntu 22.04 on a UMultipass VM. Let's launch it:
+
+```
+multipass launch --name expert-elf
+```
+
+> **Note**: This step can take a few minutes. It's normal and expected.
+
+### 3. Generate a worker node token from the control plane node
+
+> **Note:** Make sure your cluster is bootstrapped. See [Getting Started](getting-started.md) for instructions.
+
+Canonical Kubernetes allows you to create two types of nodes: control plane 
+and worker nodes. In this example, we'll be creating a worker node.
+
+Let's get the token that will allow our worker node to join our cluster.
+
+Run this command from your control plane node:
+
+```
+sudo k8s add-node expert-elf --worker
+```
+
+**Note**: It's best to give the new node the same name as the hostname of the 
+worker node (in this case the VM).
+
+A base64 string should be printed to your terminal. Keep it close as you 
+will need it in a few steps.
+
+### 4. Join the cluster from the other node
+
+Open a shell to the worker node:
+
+```
+multipass shell expert-elf
+```
+
+The VM doesn't come with the `k8s` snap, so let's install it.
+
```

Comment:
I think we should be careful to make sure that they launch a node running the same k8s as the existing one, and perhaps that should be explained.